### ISSUE TITLE
Details of discourse

### ISSUE THREAD

**Issue by kjetilk** [scienceai/scholarly.vernacular.io#30]
I belong to the school that prefers to be free to choose my ontologies, but I can see that the world doesn't really follow me there. 
Anyway, where I feel that scientific discourse really has something significant to gain is when you enable discourse itself to be expressed in structured form. 
There is a tiny start in my Linked Research paper:
file:///home/kjetil/fag/kjekje-phd/epistemology/noise-essay.html if you look in the source code, you'll find stuff like ```cito:repliesTo```, ```cito:disputes```, etc. The CITO ontology is pretty nice, that is something I would like to see adopted.

**Comment by darobin** [comment]
I think that we need to distinguish between a baselines interoperability framework that basically means that I can write a tool that produces SH and you can write a tool that consumes it, and they'll work together with no magic. That means being pretty strict in sticking to relatively limited semantics *that are guaranteed to be understood*. It also means that relying on indirection as may be provided through schemata is problematic.

It does not however mean that people can't use extensions on top! I see no reason why you couldn't use CITO in SH. It just means that my consuming tool might not understand it — but that's okay.

We've started a group to talk about this if you're interested: https://www.w3.org/community/scholarlyhtml/

**Comment by kjetilk** [comment]
Right, and I see that you may need to be restrictive to find adoption. And I trust your experience in this respect. However, I'm just impatient, what is the purpose of all this? Is it something that is guaranteed to be understood by a consumer that wants to produce proceedings on dead trees? Something that can port LaTeX-type scholarly discourse to the Web? Then, excuse me while I go *yawn* :-P  I feel strongly that there has to be something more in it for the researchers to make them consider not doing it the way it has been done hitherto [sic!]
